boot/initramfs_test.go

```diff
@@ -551,6 +551,13 @@ func (s *initramfsSuite) TestInitramfsRunModeChooseSnapsToMount(c *C) {
 
 			if t.blvars != nil {
 				c.Assert(bl.SetBootVars(t.blvars), IsNil, comment)
+				emptyMap := make(map[string]string, len(t.blvars))
```

Comment:
the name emptyMap is a bit confusing here, it's more resetting values than being empty?

Reply:
I renamed to `cleanBootVars`